doc/FREQUENTLY_MADE_SUGGESTIONS.md

```diff
@@ -241,17 +245,19 @@ When this happens (which will be a while, it’s pretty invasive and complicated
 
-It's not a valid idea to bring it back for a number of reasons.  One, the very large scale map destruction it would require would be a lot of work to implement, for incredibly low tangible benefit. Two, the feasibility of a survivor breaking into a nuclear silo and successfully launching a missile is negligible, the most likely situation is that the silo would have been put into some kind of lockdown or even destroyed once the staff deemed it infeasible to continue standing by, rendering a launch literally impossible, but even if that didn't happen, navigating the security mechanisms and failsafes the launch system would be expected to have is so difficult as to be effectively impossible. Three, the impact of an ICBM launch is the wrong scale for the game, if the desired feature is "destroy an area 150m across from a distance", which is what the previous feature amounted to, there are numerous options for achieving that without the ludicrous and impossible overkill embodied in a nuclear strike.  For example, an artillery or even mortar barrage has the capability of leveling a 100m or even larger area, and is a million times more feasible to acquire than a nuclear launch.
+It's not a valid idea to bring it back for a number of reasons.  One, the very large scale map destruction it would require would be a lot of work to implement, for incredibly low tangible benefit. Two, the feasibility of a survivor breaking into a nuclear silo and successfully launching a missile is negligible, the most likely situation is that the silo would have been put into some kind of lockdown or even destroyed once the staff deemed it infeasible to continue standing by, rendering a launch literally impossible. Even if that didn't happen, navigating the security mechanisms and failsafes the launch system would be expected to have is so difficult as to be effectively impossible, and that assumes that the launch systems are still accessible let alone operational. Three, the impact of an ICBM launch is the wrong scale for the game, if the desired feature is "destroy an area 150m across from a distance", which is what the previous feature amounted to, there are numerous options for achieving that without the ludicrous and impossible overkill embodied in a nuclear strike.  For example, an artillery or even mortar barrage has the capability of leveling a 100m or even larger area, and is a million times more feasible to acquire than a nuclear launch.
 
 Code for ICBM launch is restored (with several bugfixes) and used in No Hope mod which is shipped with the game.
 
 #### Add a lance charge for massive damage bonuses: yes, but not the way most people imagine it
 
-The general understanding of how lances work is badly warped by depictions of jousting tournaments in books and movies, dungeons and dragons and video games.
+The general understanding of how lances work is badly warped by depictions of jousting tournaments in books and movies, dungeons and dragons, and video games.
```

Comment:
> dungeons and dragons

This cannot be used as a genericized phrase, as D&D is still strong as a brand name. Thus, it should be spelled with both nouns capitalized and with an ampersand (`Dungeons & Dragons`), as per the game's [Wikipedia page](https://en.wikipedia.org/wiki/Dungeons_%26_Dragons).

Alternatively, this could be replaced with `tabletop roleplaying games`, given that D&D is likely not the only offender here.

Reply:
There is no requirements to adhere to their specific styling. Brand name or not, this isn't a trademark use.

Reply:
Let's read this as an onlooker would.

> "books and movies"... Okay, yeah, I'm with you, I've seen jousting portrayed in those. "dungeons and dragons"... What the hell? Dragons? Like, komodo dragons or *dragons* dragons? And what do dungeons have to do with it?

And before you tell me I'm viewing the phrasing too literally for the sake of taking a jab: "books and movies" and "dungeons and dragons" form a list of dual entities which, when spelled in lowercase, can only refer to the actual objects listed (books, movies, dungeons, dragons). Extra cultural knowledge is required to piece together what "dungeons and dragons", lowercase, is supposed to mean.

To make this make perfect sense, all you need to do is replace two letters.

Reply:
Ultimately such editorializing is outside of the changes I've made, and not within the scope of what I'm trying to do. If you feel strongly about this I invite you to open a PR to change it, but it's not in scope for this PR. Thanks.